bot/resources/evergreen/trivia_quiz.json

```diff
@@ -372,7 +372,7 @@
     {
       "id": 216,
       "question": "In set builder notation, {p/q | q ≠ 0, p & q ∈ Z} represents what?",
-      "answer": "Rational Numbers"
+      "answer": "rationals, Rational Numbers"
```

Comment:
Maybe capitalise this for consistency, check for capitalisation in other areas as ive noticed this before.